### PR TITLE
[MS] Added about page and modal

### LINKED ISSUE #4977: Add About page

### PR DESCRIPTION
**Pull request by Max-7**
Closes #4977 

- [X] Keep changes in the pull request as small as possible
   
- [X] Ensure the commit history is sanitized
   
- [X] Give a meaningful title to your PR
- [X] Describe your changes
   
- [X] Link any related issue in the description

### CODE REVIEW

File: client/src/locales/en-US.json [hunk 1]
```diff
@@ -393,5 +403,20 @@
...
+        "appInfo": {
+            "version": "Version",
+            "developper": "Developper",
+            "license": "License",
+            "project": "Project"
+        },
+        "update": {
+            "title": "Updates",
+            "upToDate": "Your software is up-to-date.",
+            "notUpToDate": "Your software is not up-to-date.",
```
Comment (fabienSvtr) [inline client/src/locales/en-US.json:417]: Change it in positive way: "A new version is available"

File: client/src/locales/fr-FR.json [hunk 2]
```diff
@@ -393,5 +403,20 @@
...
+    "AboutPage": {
+        "appInfo": {
+            "version": "Version",
+            "developper": "Développeur",
+            "license": "Licence",
+            "project": "Project"
+        },
+        "update": {
+            "title": "Mise à jour",
+            "upToDate": "Votre logiciel est à jour.",
```
Comment (fabienSvtr) [inline client/src/locales/fr-FR.json:416]: Same but in French: "Une nouvelle version est disponible"

**Review by fabienSvtr — CHANGES_REQUESTED** [review]

File: client/src/locales/en-US.json [hunk 3]
```diff
@@ -1,4 +1,13 @@
 {
+    "app": {
+        "name": "Parsec",
+        "developper": "Scille",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:4]:
```suggestion
        "developer": "Scille",
```

File: client/src/locales/en-US.json [hunk 4]
```diff
@@ -1,4 +1,13 @@
...
+        "developperLink": "https://scille.eu",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:5]:
```suggestion
        "developerLink": "https://scille.eu",
```

File: client/src/locales/en-US.json [hunk 5]
```diff
@@ -393,5 +403,20 @@
             "actionDetails": "View detail",
             "actionSeeCommonWorkspaces": "See common workspaces"
         }
+    },
+    "AboutPage": {
+        "appInfo": {
+            "version": "Version",
+            "developper": "Developper",
```
Comment (NicoTuxx) [inline client/src/locales/en-US.json:410]:
```suggestion
            "developer": "Developer",
```

File: client/src/locales/fr-FR.json [hunk 6]
```diff
@@ -1,4 +1,13 @@
 {
+    "app": {
+        "name": "Parsec",
+        "developper": "Scille",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:4]:
```suggestion
        "developer": "Scille",
```

File: client/src/locales/fr-FR.json [hunk 7]
```diff
@@ -1,4 +1,13 @@
...
+        "developperLink": "https://scille.eu",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:5]:
```suggestion
        "developerLink": "https://scille.eu",
```

File: client/src/locales/fr-FR.json [hunk 8]
```diff
@@ -393,5 +403,20 @@
             "actionDetails": "Voir le détail",
             "actionSeeCommonWorkspaces": "Voir les espaces de travail communs"
         }
+    },
+    "AboutPage": {
+        "appInfo": {
+            "version": "Version",
+            "developper": "Développeur",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:410]:
```suggestion
            "developer": "Développeur",
```

File: client/src/locales/fr-FR.json [hunk 9]
```diff
@@ -393,5 +403,20 @@
...
+            "license": "Licence",
+            "project": "Project"
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:412]:
```suggestion
            "project": "Projet"
```

File: client/src/views/about/AboutModal.vue [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+      </ion-buttons>
+    </ion-header>
+    <!-- content -->
+    <ion-content>
+      <about-view />
+    </ion-content>
+  </ion-page>
+</template>
+
+<script setup lang = "ts" >
```
Comment (NicoTuxx) [inline client/src/views/about/AboutModal.vue:36]:
```suggestion
<script setup lang="ts">
```

File: client/src/views/about/AboutView.vue [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+          {{ $t('AboutPage.appInfo.version') }}
+        </ion-label>
+        <ion-text class="app-info-value">
+          {{ $t('app.name') }} v{{ getAppVersion() }}
+        </ion-text>
+      </ion-item>
+
+      <ion-item>
+        <ion-label class="app-info-key">
+          {{ $t('AboutPage.appInfo.developper') }}
```
Comment (NicoTuxx) [inline client/src/views/about/AboutView.vue:26]:
```suggestion
          {{ $t('AboutPage.appInfo.developer') }}
```

File: client/src/views/about/AboutView.vue [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+        </ion-label>
+        <ion-text class="app-info-value">
+          <a
+            :href="$t('app.developperLink')"
```
Comment (NicoTuxx) [inline client/src/views/about/AboutView.vue:30]:
```suggestion
            :href="$t('app.developerLink')"
```

File: client/src/views/about/AboutView.vue [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+            target="_blank"
+          >
+            {{ $t('app.developper') }}
```
Comment (NicoTuxx) [inline client/src/views/about/AboutView.vue:33]:
```suggestion
            {{ $t('app.developer') }}
```

File: client/tests/e2e/specs/test_about_modal.ts [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+    cy.dropTestbed();
+  });
+
+  it('Opens the settings dialog', () => {
+    cy.get('.about-modal').should('exist');
+    cy.get('.about-modal').get('.title-h2').contains('About');
+    cy.get('.about-modal').find('.app-info').find('.app-info-key').as('keys').should('have.length', 4);
+    cy.get('.about-modal').find('.app-info').find('.app-info-value').as('values').should('have.length', 4);
+    cy.get('@keys').eq(0).contains('Version');
+    cy.get('@keys').eq(1).contains('Developper');
```
Comment (NicoTuxx) [inline client/tests/e2e/specs/test_about_modal.ts:20]:
```suggestion
    cy.get('@keys').eq(1).contains('Developer');
```

File: client/tests/e2e/specs/test_about_page.ts [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+  afterEach(() => {
+    cy.dropTestbed();
+  });
+
+  it('Opens the about page', () => {
+    cy.get('.topbar-left__title').find('.title-h2').contains('About');
+    cy.get('.about-container').find('.app-info-key').as('keys').should('have.length', 4);
+    cy.get('.about-container').find('.app-info-value').as('values').should('have.length', 4);
+    cy.get('@keys').eq(0).contains('Version');
+    cy.get('@keys').eq(1).contains('Developper');
```
Comment (NicoTuxx) [inline client/tests/e2e/specs/test_about_page.ts:20]:
```suggestion
    cy.get('@keys').eq(1).contains('Developer');
```

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

**Review by NicoTuxx — APPROVED** [review]

**Review by fabienSvtr — APPROVED** [review]

**Pull request merged**